animations/test/misc/hero_animation_test.dart

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:animations/src/misc/hero_animation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+Widget createHeroAnimationDemoScreen() => MaterialApp(
+      home: HeroAnimationDemo(),
+    );
+
+void main() {
+  group('Hero Animation Tests', () {
+    testWidgets('Size of Container changes on Tap', (tester) async {
+      await tester.pumpWidget(createHeroAnimationDemoScreen());
+
+      // Get the initial Container.
+      final initialContainer =
```

Comment:
This isn't actually getting the size of the Container. It's getting the constraints the Container will use when sizing its child, which is a slightly different property. See `expand_card_test.dart` for an example of how to get a widget's size during testing.

Reply:
Fixed this in new commit.